packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -197,9 +197,10 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       classNames.footer,
       {
         display: 'flex',
+        alignItems: 'center',
+        color: palette.white,
         selectors: {
-          // TODO: global class name usage should be converted to a button styles function once Button supports JS styling
-          [`.${classNames.button}:not(:first-child)`]: {
```

Comment:
Note: I changed the selector to target all direct children of `footer` to avoid having to repeat `marginLeft: '20px'` for the buttons and for the custom footer. I can revert if appropriate